### PR TITLE
add sky background calculation using rubin_sim.skybrightness code

### PR DESCRIPTION
**Pull request by jchiang87**

### CODE REVIEW

File: imsim/sky_model.py
Comment (rmjarvis) [inline imsim/sky_model.py:113]: All the existing types so far are capitalized (CamelCase), so I would call this SkyLevel to match current conventions.

File: imsim/sky_model.py
Comment (rmjarvis) [inline imsim/sky_model.py:105]: I think `wcs = base['wcs']` should work here.  At the time when we calculate the sky level, the wcs should already be built and saved.  No need to remake it here.
Edit: given the below comment, I don't think you even need the wcs here.

File: imsim/sky_model.py
Comment (rmjarvis) [inline imsim/sky_model.py:107]: A few things here.
First, `wcs.center` is not necessarily anything we want to use here.  It's a arbitrary location that doesn't have to have any connection to the image.  I think what we want is `base['world_center']`.
Second, I would suggest passing this as a CelestialCoordinate, rather than convert to ra, dec in degrees and then in the other function have LSST sims use degrees.  That way the two uses of degrees, which need to match, are in the same function.
Finally, a slightly simpler syntax to get the angle in degrees is just `ra.deg`.

File: tests/test_sky_model.py
Comment (rmjarvis) [inline tests/test_sky_model.py:6]: This test looks fine.  But I'll note that we already had a file test_skyModel.py.  Is anything in there reusable, modifying to the new syntax?  If not we should either remove it or mark which ones are still aspirational.

**Review by rmjarvis — CHANGES_REQUESTED** [review]
Thanks for getting this working Jim.  I do have a few suggestions, but it basically looks good.

File: tests/test_sky_model.py
Comment (jchiang87) [reply tests/test_sky_model.py:6]: I was thinking of just removing test_skyModel.py since almost all of the tests are specific to how that old skyModel.py code was written, i.e., using that `NoiseAndBackgroundBase` class from sims_GalSimInterface.  So, I'll just go ahead and remove it.

**Review by rmjarvis — APPROVED** [review]

**Comment by jchiang87** [comment]
@rmjarvis Before merging, I'm thinking it might be good to move the `from rubin_sim import sky_brightness` line from the top of sky_model.py to inside the `SkyModel.__init__` function.  This way, `rubin_sim` doesn't need to be installed, and one can just set
```
input.sky_model: ""
image.sky_level:  <some value>
```
and still run the code.  What do you think?

**Comment by rmjarvis** [comment]
Yeah, that's probably a good idea.  At least until rubin_sim gets a bit easier to install.

**Comment by jchiang87** [comment]
Unless I hear otherwise, I'll merge this after 5pm PT today.

**Pull request merged**
